docs/01_Before-you-start.md

```diff
@@ -0,0 +1,15 @@
+# Before you start
+
+Before you start your study, here are some things you might want to consider:
+
+* [Reproducibility]
```

Comment:
I was thinking that until we have filled those subsections we could comment out those, no?

This way we have our structure in the markdown files but it is not displayed in the mkdocs until we put something in that section.

Alternatively we could keep those sections to show people what is our goal structure.

```
<!-- * [Reusing data]
* [Defining your terms and your task]
* [Ontologies]
* [Piloting]
* [Pre-registration]
* [Optimizing your design]
* [Design efficiency]
* [Power]
* [For MVPA: same analysis approach]
* [Defining your region of interest]
* [Using previous results]
* [Localizers]
* [Atlases]
* [Non-standard templates]
* [Stimuli presentation softwares]
 * [psychopy]
 * [expyriment]
 * [psychtoolbox]
* [Existing protocols] -->
```

Reply:
Sounds good!

Reply:
Ah no worries, I can do it! I can use the learning experience :)